apps/encryption/lib/Command/FixEncryptedVersion.php

```diff
@@ -212,18 +218,19 @@ private function verifyFileContent($path, OutputInterface $output, $ignoreCorrec
 	}
 
 	/**
-	 * @param string $path
-	 * @param OutputInterface $output
 	 * @param bool $includeZero whether to try zero version for unencrypted file
-	 * @return bool
 	 */
-	private function correctEncryptedVersion($path, OutputInterface $output, bool $includeZero = false): bool {
+	private function correctEncryptedVersion(string $path, OutputInterface $output, bool $includeZero = false): bool {
 		$fileInfo = $this->view->getFileInfo($path);
 		if (!$fileInfo) {
 			$output->writeln("<warning>File info not found for file: \"$path\"</warning>");
 			return true;
 		}
 		$fileId = $fileInfo->getId();
+		if ($fileId === null) {
+			$output->writeln("<warning>File info contains no id for file: \"$path\"</warning>");
```

Comment:
Seems more appropriate:

```suggestion
			$output->writeln("<warning>Unknown file: \"$path\"</warning>");
```

Reply:
Well, this is a case that should not happen I think, I added the test to please psalm, and in the case it happens I would prefer the warning to be quite precise to ease debugging. So yeah the message is technical, but so it the error case.